CHANGES.md

```diff
@@ -1,3 +1,9 @@
+1.5.3
+=====
+
+* Add `launchNamedPureLogWithPure` to `PureLogging`
+* Improve documentation for `launchPureLog`
```

Comment:
I think you want to say that you improved documentation for `launchNamedPureLog` function :)